scripts/get_uwsgi_args.py

```diff
@@ -99,6 +99,7 @@ def _get_uwsgi_args(cliargs, kwargs):
         'hook-master-start': ('unix_signal:2 gracefully_kill_them_all',
                               'unix_signal:15 gracefully_kill_them_all'),
         'py-call-osafterfork': True,
+        'cron': '0 -1 -1 -1 -1 python scripts/tool_shed/build_ts_whoosh_index.py -c config/tool_shed.yml --config-section tool_shed',
```

Comment:
Technically this will fail if you don't have a `config/tool_shed.yml`. I'm not super worried about that though.

Reply:
Actually... you can template `config_file` in to this to fix it.